bootstraptest/runner.rb

```diff
@@ -721,7 +721,7 @@ def check_coredump
 end
 
 def mmtk?
-  `#{BT.ruby} -e 'p defined?(GC::MMTk)'`.strip == '"constant"'
+  `#{BT.ruby} -e 'puts (defined?(GC::MMTk.enabled?) && GC::MMTk.enabled?) || false'`.strip == 'true'
```

Comment:
What about using `print` instead of `puts`? Then you won't need to strip, I think.